frontend/src/Components/SearchBar.js

```diff
@@ -1,20 +1,28 @@
-import { Card, Stack, Form, Button } from "react-bootstrap";
+import { Card, Stack, Form } from "react-bootstrap";
 import { MDBCol } from "mdbreact";
-import { Search } from "react-bootstrap-icons";
-import { useState } from "react";
+import DescriptionBox from "./DescriptionBox";
+// import { Search } from "react-bootstrap-icons";
+// import { useState } from "react";
 
 export default function SearchBar() {
   //const [description, setDescription] = useState("");
   //description should be set to the text from the search bar
-  const description = "test";
+
+  const description = document.getElementById("text_input");
   const onSubmitForm = async (e) => {
+    console.log(description.value);
     e.preventDefault();
```

Comment:
I think you should be using the useState hook to grab the query. This allows us to control the input. If you are not familiar with it, look into using state and the onChange event handler for textfields in React. By doing this there is no need to grab the element by id.